### DISCUSSION TITLE
单元测试例子可以再详细些吗

### DISCUSSION THREAD

**Discussion by Tyf2345** [hellof2e/quark-design#281] [Q&A] [answered]
- 单元测试是怎么跑起来的，run 命令是什么，是否需要配置文件；
- 我准备写 web端的 组件库，但是卡在了单元测试这块
- 希望可以出个完整的示例

**Comment by yuhaiyang1** [comment, marked as answer]
可以执行yarn run test 命令
现在每个组件有一个 index.test.js 可以参考下
